examples/example-phase1/src/constants/url.ts

```diff
@@ -0,0 +1,3 @@
+const BLOGS_URL = "http://localhost:1234/blogs";
```

Comment:
Separate into an .env and ignore it. BWT, move `blogs` into another constants call `ENDPOINTS` so that you can reuse for revalidating